lib/logs/log_formatter.rb

```diff
@@ -20,6 +20,8 @@ def initialize(data)
           when :format then 3
           when :params then 4
           when :status then 5
+          when :queries then 5.3
+          when :cached_queries then 5.6
```

Comment:
I'm introducing these floats so as not to have to increment all of the other sort values below.